src/commands/code-analyzer/rules.ts

```diff
@@ -1,19 +1,40 @@
-import {SfCommand} from '@salesforce/sf-plugins-core';
+import {Flags, SfCommand} from '@salesforce/sf-plugins-core';
+import {RulesAction} from '../../lib/actions/RulesAction';
+import {BundleName, getMessage} from '../../lib/messages';
 
-export default class Rules extends SfCommand<void> {
+export default class RulesCommand extends SfCommand<void> {
 	// We don't need the `--json` output for this command.
 	public static readonly enableJsonFlag = false;
-	public static readonly summary = 'summary boop'; // TODO
-	public static readonly description = 'description boop'; // TODO
-	public static readonly examples = [ // TODO
-		'example1 boop',
-		'example2 boop'
+	public static readonly summary = getMessage(BundleName.RulesCommand, 'command.summary');
+	public static readonly description = getMessage(BundleName.RulesCommand, 'command.description');
+	public static readonly examples = [
+		getMessage(BundleName.RulesCommand, 'command.examples')
 	];
 
-	public static readonly flags = {};
+	public static readonly flags = {
+		'rule-selector': Flags.string({
+			summary: getMessage(BundleName.RulesCommand, 'flags.rule-selector.summary'),
+			char: 'r',
+			multiple: true,
+			delimiter: ',',
+			default: ["recommended"]
+		}),
+		'config-file': Flags.file({
+			summary: getMessage(BundleName.RulesCommand, 'flags.config-file.summary'),
+			char: 'c',
+			exists: true
+		}),
+		view: Flags.string({
+			summary: getMessage(BundleName.RulesCommand, 'flags.view.summary'),
+			char: 'v',
+			options: ['table', 'detail'] // TODO: Should probably be enum?
```

Comment:
is it detail or details? I just don't remember.

Reply:
The draft doc says "detail", so I used that.